exampleCourse/courseInstances/Sp15/assessments/hw05-demoDrawingExamples/infoAssessment.json

```diff
@@ -21,7 +21,7 @@
     "zones": [
         {
             "questions": [
-                {"id": "element/drawingGallery",                 "points": 2, "maxPoints": 6},
+                {"id": "element/drawingGallery",                  "points": 2, "maxPoints": 6},
```

Comment:
I didn't go through all the demo questions carefully, but I did notice that:

1. `demo/drawing/centroid` allows the student to move the triangle and rectangle -- I think they should be fixed?
2. `demo/drawing/centroid` doesn't show the correct answer
3. `demo/drawing/vmDiagrams` doesn't show the correct answer

I don't want to hold up this PR now because of demo questions, so maybe we should add an issue about these questions and also that we should go through the other demo drawing questions?